data/campaigns/Heir_To_The_Throne/units/Battle_Princess.cfg

```diff
@@ -25,7 +25,7 @@
         {ABILITY_INITIATIVE}
     [/abilities]
     usage=fighter
-    description= _ "A noble by birth, the Princess has learnt swordplay with the greatest generals and battle tactics with the greatest sages, making her both a great combatant and a leader. Now battle-hardened and strong of will, she can aid those around her in the art of combat."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}+{SPECIAL_NOTES_INITIATIVE}
+    description= _ "A noble by birth, the Princess has learnt swordplay with the greatest generals, and battle tactics with the greatest sages, making her both a great combatant and a leader. Now battle-hardened and strong of will, she can aid those around her in the art of combat."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}+{SPECIAL_NOTES_INITIATIVE}
```

Comment:
I'm not convinced this is correct, but _if_ it is correct it should be applied to units/Princess.cfg as well.

Reply:
> I'm not convinced this is correct, but _if_ it is correct it should be applied to units/Princess.cfg as well.

I think I copied it from there. Since we had two strings, one with comma, one without.

Reply:
Actually there are _four_ strings:

```
data/campaigns/Heir_To_The_Throne/units/Princess.cfg:28:    description= _ "A noble by birth, the Princess has learnt swordplay with the greatest generals and battle tactics with the greatest sages, making her both a great combatant and a leader. The units of lower level around the Princess will fight better due to her presence."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}+{SPECIAL_NOTES_INITIATIVE}
data/campaigns/Heir_To_The_Throne/units/Princess.cfg:84:        description= _ "A noble by birth, the Princess has learnt swordplay with the greatest generals and battle tactics with the greatest sages, making her both a great combatant and a leader. The units of lower level around the Princess will fight better due to her presence."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}+{SPECIAL_NOTES_INITIATIVE}+{SPECIAL_NOTES_MAGICAL}
data/campaigns/Heir_To_The_Throne/units/Battle_Princess.cfg:28:    description= _ "A noble by birth, the Princess has learnt swordplay with the greatest generals and battle tactics with the greatest sages, making her both a great combatant and a leader. Now battle-hardened and strong of will, she can aid those around her in the art of combat."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}+{SPECIAL_NOTES_INITIATIVE}
data/campaigns/Heir_To_The_Throne/units/Battle_Princess.cfg:124:        description= _ "A noble by birth, the Princess has learnt swordplay with the greatest generals, and battle tactics with the greatest sages, making her both a great combatant and a leader. Now battle-hardened and strong of will, she can aid those around her in the art of combat."{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}+{SPECIAL_NOTES_INITIATIVE}+{SPECIAL_NOTES_MAGICAL}
```

Reply:
Hmm, then I'll have to update those four to use the same convention once somebody tells me which one to use.

Reply:
I comma usually means "take a breath here" so it works best on line 124, as @jostephd suggests.